src/components/ZAvatar/ZAvatar.vue

```diff
@@ -33,6 +33,7 @@ const SIZES = {
   xs: { classes: 'leading-none h-4 w-4 text-xxs', text: 'xs' },
   sm: { classes: 'leading-none h-6 w-6 text-xs', text: 'sm' },
   md: { classes: 'leading-none h-8 w-8 text-sm', text: 'md' },
+  base: { classes: 'leading-none h-9 w-9 text-base', text: 'base' },
```

Comment:
Ideally, h-10 should've been used here to maintain the increment order.

@yash-deepsource what do you think we should do in situations where arbitrary dimensions are required for zeal components?

Reply:
I would name the variant for the specific case, then, instead of trying to fit into the scale.  It is an escape hatch, and this scale should be well-defined instead by design. :/